katalog/etcd-backup-s3/MAINTENANCE.md

```diff
@@ -0,0 +1,5 @@
+# ETCD Backup S3 Maintenance
+In order to update and maintain the package:
```

Comment:
we should specify here that the manifests are hand-made and there is no upstream to follow.